example/timer/odp_timer_accuracy.c

```diff
@@ -197,6 +197,7 @@ static int parse_options(int argc, char *argv[], test_opt_t *test_opt)
 
 	memset(test_opt, 0, sizeof(*test_opt));
 
+	test_opt->cpu_count = 1;
```

Comment:
This changes the behavior of the test in make check and in CI. Maybe odp_timer_accuracy_run.sh should now explicitly specify -c 0 option to retain the old behavior when the test is run through the script..

Reply:
This -c option was just added in Oct. So, the test has been single threaded in CI for long time, and has now accidentally run on all threads for two months.